src/components/molecules/NumberText/NumberText.tsx

```diff
@@ -30,7 +30,7 @@ interface StyleProps {
    * Where the rendered text should be aligned to.
    * @default 'center'
    */
-  align?: 'left' | 'right' | 'center';
+  align?: 'left' | 'right' | 'center' | 'flex-start' | 'flex-end';
```

Comment:
```suggestion
  align?: 'left' | 'right' | 'center';
```
Flex-start and flex-end isn't needed as an option if left and right is mapped to these